server/core/templates/admin/admin_group_view_partial.html

```diff
@@ -0,0 +1,116 @@
+                        type="button" class="btn btn-danger">Remove</button>
+                (% endif %)
+            </div>
+        </li>
+        (% endfor %)
+        (% if can_edit %)
+            <li class="list-group-item"  id="addEmailField">
+                <div class="input-group d-flex flex-row justify-content-between">
+                    <input type="email" name="mail" class="form-control" id="groupAddEmailInput" placeholder="Group's Email address" aria-label="Group's Email address" aria-describedby="groupAddEmailBtn">
+                    <button id="groupAddEmailBtn" class="btn btn-success text-nowrap"
+                            hx-post="/ui/admin/api/group/(( group.uuid ))/mail"
+                            hx-include="#groupAddEmailInput"
+                            hx-target="#addEmailField"
+                            hx-swap="beforebegin"
+                            type="button">
+                        Add Email
+                    </button>
+                </div>
+                <div class="invalid-feedback">
+                    Please choose a valid email.
+                </div>
+            </li>
+        (% endif %)
+    </ol>
+(% endif %)
+</form>
+
+<label class="mt-3 fw-bold">Account Policy (ACP)</label>
+<div>
+(% if group.acp.enabled %)
+    Nothing here yet 🙈🙈
+(% else %)
+    Currently disabled.
+    (% if can_edit %)
+        <button hx-post="/ui/api/group/(( group.uuid ))/enable_acp" hx-confirm="Currently no known way to disable again, ok to continue." class="btn btn-success d-block">Enable</button>
+    (% endif %)
+(% endif %)
+</div>
+
+<label class="mt-3 fw-bold">Members</label>
+<form hx-validate="true" hx-ext="bs-validation">
+(% if group.members.len() == 0 %)
+    <p>none</p>
+(% else %)
+    <ul class="list-group col-12 col-md-8 col-lg-6">
+        (% let group_uuid = group.uuid.clone() %)
+        (% for member in group.members %)
+            (% include "admin_group_member_partial.html" %)
+        (% endfor %)
+        (% if can_edit %)
+        <li class="list-group-item" id="addMemberField">
+            <div class="input-group d-flex flex-row justify-content-between">
+                <input type="text" class="form-control" id="groupAddMemberInput" placeholder="Member name/spn/uuid" name="member" aria-label="Member name/spn/uuid to add as group member" aria-describedby="groupAddMemberBtn">
+                <button id="groupAddMemberBtn" class="btn btn-success text-nowrap"
+                        hx-post="/ui/admin/api/group/(( group.uuid ))/member"
+                        hx-include="#groupAddMemberInput"
+                        hx-target="#addMemberField"
+                        hx-swap="beforebegin"
+                        type="button">
+                    Add Member
+                </button>
+            </div>
+            <div class="invalid-feedback">
+                Please choose a valid email.
```

Comment:
see previous suggestion about wording

Reply:
Should the `Please choose a valid POSIX group ID.` then also become `Please enter a valid POSIX group ID.` to be consistent